lisa/analysis/load_tracking.py

```diff
@@ -211,8 +214,8 @@ def plotter(axes, local_fig):
                 axis.set_title('CPU{}'.format(cpu))
                 df = cpus_df[cpus_df["__cpu"] == cpu]
 
-                df[['util']].plot(ax=axis, drawstyle='steps-post', alpha=0.4)
-                df[['load']].plot(ax=axis, drawstyle='steps-post', alpha=0.4)
+                for signal in signals:
+                    df[[signal]].plot(ax=axis, drawstyle='steps-post', alpha=0.4)
```

Comment:
isn't the above just:
```
df[[signals]].plot(ax=axis, drawstyle='steps-post', alpha=0.4)
```

i.e. `signals` passed to `df` directly

Reply:
I can give it a try